src/site/asn1.rst

```diff
@@ -62,6 +62,15 @@ country ``name`` (string), ``population`` (int), ``area`` (double), and ``contin
 
     ``"Canada, North America, 34207000, 9976140.00"``
 
+#. Write an ``equals`` method to compare two (2) country objects. Simply check if all fields are the same --- for our purposes, if they are, then we'll say they are equal.
```

Comment:
Need to be careful here. Make sure they're overriding `Object#equals`; ie, the parameter is an `Object`. They'll have to do some casting -- you may want to write the skeleton for them?

Reply:
Damn, good catch my dude.

Reply:
I ended up pointing them to the aside on equals in topic3.